docs/README.md

```diff
@@ -57,16 +57,30 @@ You can download the Version 1.x.x Zowe documentation in PDF format from the lin
 **[V1.1.0](https://docs.zowe.org/v1-1-x/Zowe_Documentation.pdf)** |
 **[V1.0.x](https://docs.zowe.org/v1-0-x/Zowe_Documentation.pdf)**
 
-### Zowe CLI command reference guide
+### Zowe CLI command reference guides
 
 View detailed documentation on commands, actions, and options in Zowe CLI. The reference document is based on the `@zowe-v1-lts` version of the CLI. A PDF document, an interactive online version, and a ZIP file containing the HTML for the online version:
 
 <p class="action">
-<a href="./web_help/index.html" target="_blank" class="nav-link action-button"> <span>Browse Online</span></a>
+<a href="./web_help/index.html" target="_blank" class="nav-link action-button"> <span>Browse CLI Reference (Online)</span></a>
 <a href="./CLIReference_Zowe.pdf" class="nav-link action-button"> <span>Download CLI Reference (PDF)</span></a>
 <a href="./zowe_web_help.zip" class="nav-link action-button"> <span>Download CLI Reference (ZIP)</span></a>
 </p>
 
+### Zowe Client SDK reference guides
+
+Refer to the following Zowe Client SDK reference guides for information about the API endpoints:
+
+<p class="action">
+<a href="./typedoc/globals.html" target="_blank" class="nav-link action-button"> <span>Browse Node SDK Reference (Online)</span></a>
+<a href="./zowe-node-sdk-typedoc.zip" class="nav-link action-button"> <span>Download Node SDK Reference (ZIP)</span></a>
+</p>
+
+<p class="action">
+<a href="" target="_blank" class="nav-link action-button"> <span>Browse Python SDK Reference (Online)</span></a>
+<a href="./CLIReference_Zowe.pdf" class="nav-link action-button"> <span>Download Python SDK Reference (ZIP)</span></a>
+</p>
```

Comment:
Do we want to update this hrefs?

Reply:
Those need to be updated. Otherwise, clicking the Python SDK Reference download button just downloads the Node CLI Reference, and clicking the Browse Python SDK Reference button just redirects to the top of the home page.

Reply:
Do you know what the file names will be to link to? I'm not sure how to hook up the links.

Reply:
Browse online Python SDK: `https://zowe-client-python-sdk.readthedocs.io/en/latest/index.html`
Download Python SDK (PDF): `https://zowe-client-python-sdk.readthedocs.io/_/downloads/en/latest/pdf/`

Reply:
That is correct @zFernand0